.circleci/config.yml

```diff
@@ -359,6 +359,26 @@ jobs:
       - store_artifacts:
           path: /go/src/github.com/tendermint/tendermint/tendermint-*.tar.gz
 
+  # Get Node.js and test REST implementation against the swagger documentation of the SDK version used
+  contract_tests:
+    <<: *defaults
+    steps:
+      - attach_workspace:
+          at: /tmp/workspace
+      - restore_cache:
+          key: v4-pkg-cache
+      - restore_cache:
+          key: v3-tree-{{ .Environment.CIRCLE_SHA1 }}
+      - checkout
+      - setup_remote_docker:
+          docker_layer_caching: true
+      - run:
+          name: Contract tests
+          command: |
+            go get github.com/snikch/goodman/cmd/goodman
+            export PATH=~/.local/bin:$PATH
+            ./scripts/get_nodejs.sh && make contract-tests
```

Comment:
fetching node.js on every build seems like an overkill

Reply:
maybe we can just use https://circleci.com/docs/2.0/language-javascript/ node.js container?

Reply:
Here it's not too nice, but we have to use either our `go` container and add the `nodejs` binary ~13MB or the other way around, use the `node` container and add `go` to build the two binaries we use for this tool:
- tendermint
- hooks

Reply:
right, so we can't use nodejs container because we need tendermint binary